src/Query.ts

```diff
@@ -18,12 +19,22 @@ export type SortingProperty =
     | 'tag';
 type Sorting = { property: SortingProperty; reverse: boolean };
 
+export type GroupingProperty =
+    | 'filename'
+    | 'folder'
+    | 'heading'
+    | 'linktext'
```

Comment:
@schemar I'm not sure about this name `linktext`... It groups by `filename > heading name`
It's similar but not consistent with `hide backlink` which displays the same text, but also contains a hyperlink.
I chose `linktext` for the name for the worst of reasons, namely that this is the name of the method on Task!

If the absence of hyperlink doesn't matter, maybe it would be better to do `group by backlink` ??

Reply:
Technically, it's not a backlink. It's a link. I have no idea why I called those backlinks in the first place 😔 

Therefore, I think any name like linktext is fine? Or would rather have it wrong, but consistent? I see both as valid.

Reply:
Thanks! I've hummed and hahed... and gone for backlink, so wrong but consistent.